docs/source/contribute.rst

```diff
@@ -0,0 +1,47 @@
+.. _contribute:
+
+Contributing to Cartopy
+=======================
+
+Cartopy is an open-source project that welcomes contributions from anyone. A good place
+to start contributing is by fixing typos, adding examples, or improving the explanations
+of the functions.
+
+To contribute new features or bug fixes, you can create a pull request with your code changes
+on GitHub. The easiest way to get started contributing code is to clone the repository to your
+local system and install the conda development environment included with the source code.
+
+.. code::
+
+  git clone git@github.com:YOUR-USERNAME/cartopy.git
```

Comment:
Do we need to explain how to fork to their account above first?

Reply:
I'm not sure what the best way to do that is? I added text and a link to `/scitools/cartopy/fork`, which I think will get people started...